rclcpp/src/rclcpp/utilities.cpp

```diff
@@ -89,14 +93,11 @@ signal_handler(int signal_value)
 void
 rclcpp::utilities::init(int argc, char * argv[])
 {
-  (void)argc;
-  (void)argv;
   g_is_interrupted.store(false);
-  rmw_ret_t status = rmw_init();
```

Comment:
`rcl_init` does not call `rmw_init`, I'll open a pull request for this:

https://github.com/ros2/rcl/pull/69

However, there is no `rmw_shutdown` right now and `rmw_init` is generally not used, so I think I probably didn't use it because I knew we had to change some things about how init/shutdown worked with the rmw interface anyways. I'm not sure it should be merged in it's asymmetric state, or if we should address this when we fix the rmw init/shutdown code.